prow/config/prow-config-documented.yaml

```diff
@@ -472,139 +472,10 @@ owners_dir_blacklist:
     repos:
         "": null
 plank:
-    # DefaultDecorationConfigs holds the default decoration config for specific values.
```

Comment:
Don't we still want to keep the documentation for these fields?

Reply:
This is autogenerated and I don't think it can be easily fixed since the field now has type `json.RawMessage`. I don't think there is a good way to display 2 alternative formats or a good way to indicate that to the generator.